md/deploy-bundle.md

```diff
@@ -11,7 +11,7 @@
 * `Tomcat-`_`version`_: a folder/file structure to be merged with an existing setup of Apache Tomcat, in order to install Bonita BPM Portal and Bonita BPM Engine.
 * `Wildfly-`_`version`_: a folder/file structure to be merged with an existing setup of WildFly in order to install Bonita BPM Portal and Bonita BPM Engine.
 <a id="platform_setup_tool" />
-* `setup`: a command-line tool that creates the Bonita BPM platform before it can be run. It creates the Bonita BPM platform internal database, and stores the runtime configuration.
+* `setup`: a command-line tool that creates the Bonita BPM Platform before it can be run. It creates the Bonita BPM Platform internal database, and stores the runtime configuration.
```

Comment:
Sure enough, the rendering looks better, there. :-)
Thanks.
(You can ignore my original comment.)